src/pages/stacks-blockchain/running-testnet-node.md

```diff
@@ -52,26 +52,35 @@ git clone https://github.com/blockstack/stacks-blockchain.git; cd stacks-blockch
 Install the Stacks node by running:
 
 ```bash
-cargo build --workspace --bin stacks-node
-# binary will be in target/debug/stacks-node
-./target/debug/stacks-node krypton
-# release build
 cargo build --workspace --release --bin stacks-node
 # binary will be in target/release/stacks-node
 ```
+To install Stacks node with extra debugging symbols, run:
```

Comment:
would suggest a new line here right after the code sample - I believe that is one of our linting rules

Reply:
@agnelvishal : 
`yarn lint:prettier:fix` should resolve any linting issues, followed by a commit of any updated files

Reply:
@wileyj Ran lint. Check if it is okay